docs/atomic-update-container.1.md

```diff
@@ -0,0 +1,34 @@
+% ATOMIC(1) Atomic Man Pages
+% Giuseppe Scrivano
```

Comment:
As stated above, I think it would be better to expand on the atomic update command

atomic update --container or something like that.